queue_services/entity-filer/src/entity_filer/worker.py

```diff
@@ -217,6 +223,15 @@ async def process_filing(filing_msg: Dict, flask_app: Flask):  # pylint: disable
                 elif filing.get('changeOfRegistration'):
                     change_of_registration.process(business, filing_submission, filing, filing_meta)
 
+                elif filing.get('restoration'):
+                    if business.state == Business.State.HISTORICAL:
+                        AccountService.update_entity(
+                            business_registration=business.identifier,
+                            business_name=business.legal_name,
+                            corp_type_code=business.legal_type,
+                            state=Business.State.ACTIVE
+                        )
+
```

Comment:
I don't think this code is needed now since we haven't implemented `restoration` filing in lear. @lekshmimallika-aot thoughts?

Reply:
Ya, This is not really needed atm.

Reply:
Taking it out.